<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineFileManagerClass</class>
 <widget class="QMainWindow" name="OfflineFileManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OfflineFileManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="updateButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(204, 204, 204);
image: url(:/OfflineFileManager/res/icons/refresh_update_icon_142975.png);



</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addFolderButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(204, 204, 204);
image: url(:/OfflineFileManager/res/icons/add-folder.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="homeButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(204, 204, 204);
image: url(:/OfflineFileManager/res/icons/house.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="upButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(204, 204, 204);
image: url(:/OfflineFileManager/res/icons/upload.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addressLine"/>
       </item>
       <item>
        <widget class="QToolButton" name="searchButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(204, 204, 204);
image: url(:/OfflineFileManager/res/icons/172546_search_icon.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="separ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="fileSystemTree"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionCreate_Virtual_Folder"/>
    <addaction name="actionReturn_Home"/>
    <addaction name="actionReturn_Upper"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QMenu" name="menuRegime">
     <property name="title">
      <string>Regime</string>
     </property>
     <addaction name="actionExternal_drives"/>
     <addaction name="actionAll_drives"/>
    </widget>
    <addaction name="menuRegime"/>
    <addaction name="actionMaximum_depth"/>
    <addaction name="actionSet_Filters"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <widget class="QMenu" name="menuConnect">
     <property name="title">
      <string>Connect</string>
     </property>
     <addaction name="actionGoogle_Drive"/>
    </widget>
    <addaction name="menuConnect"/>
   </widget>
   <widget class="QMenu" name="menuUtils">
    <property name="title">
     <string>Utils</string>
    </property>
    <addaction name="actionrobocopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuUtils"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen_in_file_explorer">
   <property name="text">
    <string>Open in file explorer</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExternal_drives">
   <property name="text">
    <string>External drives</string>
   </property>
  </action>
  <action name="actionAll_drives">
   <property name="text">
    <string>All drives</string>
   </property>
  </action>
  <action name="actionMaximum_depth">
   <property name="text">
    <string>Maximum depth</string>
   </property>
  </action>
  <action name="actionGoogle_Drive">
   <property name="text">
    <string>Google Drive</string>
   </property>
  </action>
  <action name="actionrobocopy">
   <property name="text">
    <string>robocopy</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionCreate_Virtual_Folder">
   <property name="text">
    <string>Create Virtual Folder</string>
   </property>
  </action>
  <action name="actionReturn_Home">
   <property name="text">
    <string>Return Home</string>
   </property>
  </action>
  <action name="actionReturn_Upper">
   <property name="text">
    <string>Return Upper</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSet_Filters">
   <property name="text">
    <string>Set Filters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OfflineFileManager.qrc"/>
 </resources>
 <connections/>
</ui>
